oid/gen.go

```diff
@@ -11,6 +11,7 @@ import (
 	"os/exec"
 
 	"database/sql"
```

Comment:
Hmm.  goimports doesn't move it up for me, but it doesn't move it down either if I move it up manually.  I do agree that it's more consistent if it's moved up.  Will fix, thanks.